containers/home.tsx

```diff
@@ -1,66 +1,92 @@
-import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+
+// Components
+import ListCardVideo from '../components/listCardVideo/listCardVideo';
+import ArrowPrevSVG from '../components/svg/arrowPrevSVG';
+import ArrowNextSVG from '../components/svg/arrowNextSVG';
+
+// Style
+import styles from '../styles/Home.module.css';
+interface iVideo {
+  cards?: [{ title: string; alt: any; src: any }];
+}
+
+const Home: React.FC<iVideo> = () => {
+  const [viewNextVideo, setViewNextVideo] = useState(0);
+  const [showVideos, setShowVideos] = useState([]);
```

Comment:
The name of this state variable makes me think that it should be a `boolean`. What do you think?

Reply:
I think that we could rename the state but I'm not sure of know like to name it, I think that it's well for me, but maybe it looks like that's a boolean or for example displayNewVideos / displayNewVideos, how do you think it would be better?

Reply:
How about something like `videosToShow`?

Reply:
I agree with you, thank you!